scripts/data/tensorflow.ts

```diff
@@ -157,7 +153,92 @@ void* GPUcudaMallocAllocator::AllocateRaw(size_t alignment, size_t num_bytes) {
 		return nullptr;
 	#endif  // GOOGLE_CUDA
 	}
-`
-		}
+`,
+		},
+		{
+			path: "tensorflow/tensorflow/examples/speech_commands/accuracy_utils.py",
```

Comment:
Awesome work, maybe it's just more readable if we remove the first part of the path, because GitHub puts the name of the repo there, so the actual folder is just: "/tensorflow/examples/..."

Reply:
Hello Marko, I have removed the repo name, thank you for pointing it out.